squad/core/notification.py

```diff
@@ -113,8 +113,11 @@ def __send_notification__(project, notification):
         context=context,
     )
     sender = "%s <%s>" % (settings.SITE_NAME, settings.EMAIL_FROM)
-    for r in recipients:
-        message = EmailMultiAlternatives(subject, text_message, sender, [r.email])
-        if r.html:
-            message.attach_alternative(html_message, "text/html")
-        message.send()
+
+    emails = [r.email for r in recipients]
+    use_html = all([r.html for r in recipients])
```

Comment:
I'm not sure how that works on mailing lists but what you suggest creates different message IDs. Would that change the threading depending on where the reply comes from? So if there are 2 mailing lists in reply-to, would it matter from which I reply? The lists would receive messages with different IDs. That was the main concern when we discussed this solution with Antonio.